<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingToolBarWeiget</class>
 <widget class="QWidget" name="SettingToolBarWeiget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="baudRateLabel">
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>125</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14400</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="stopBitLabel">
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>125</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBox_2">
     <item>
      <property name="text">
       <string>1位</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.5位</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2位</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="verifyLabel">
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>125</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboBox_3">
     <item>
      <property name="text">
       <string>无</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>奇校验</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>偶校验</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
